### DISCUSSION TITLE
Changing Header Logo Size

### DISCUSSION THREAD

**Discussion by gigggolore** [aemi-dev/aemi#21] [Ideas]
Hi Guillaume,

i would resize the Logo for AEMI, but changes in AEMI.CSS under Site-Logo seems no effect.

My opinion is a Logo-Size like this: [https://www.valokuvataiteenmuseo.fi/en](url)

Could you give me an advice?

Thank you

**Comment by g-cqd** [comment]
Hi @gigggolore 

Sorry for my late answer, I was a bit busy.

Actually, Aemi is not designed to make this possible. There's quite a lot of things that constraint and ensure this to not happen so I can not give you a solution for this today.

I don't tell you this is not going to happen ever but not anytime soon.

Sorry for such a disappointing news 😅

**Comment by gigggolore** [comment]
Thank you for your answer! I think i will set up an testingpage and play with CSS. Maybe there is an way to get larger Logodesigns. Mine ist 40px waht ist really small.
